kolibri/plugins/management/assets/src/views/class-edit-page/user-remove-modal.vue

```diff
@@ -72,13 +70,35 @@
         required: true,
       },
     },
+    computed: {
+      formattedDeleteConfirmation() {
+        return this.$tr(
+          'deleteConfirmation',
+          {
+            username: this.bold(this.username),
```

Comment:
Made bold-ing a method only because it's used in all of these strings. Might want to locally scope it?